src/CmsManager/CmsManagerSelector.php

```diff
@@ -28,7 +25,7 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-final class CmsManagerSelector implements CmsManagerSelectorInterface, LogoutHandlerInterface
+final class CmsManagerSelector implements CmsManagerSelectorInterface
```

Comment:
We will need to do some dirty tricks with this one. It could be a good time to refactor this class too.

Reply:
I did something to fix this.